pkgs/by-name/je/jellyseerr/package.nix

```diff
@@ -54,6 +54,11 @@ stdenv.mkDerivation (finalAttrs: {
     runHook postBuild
   '';
 
+  postBuild = ''
+    # Clean up broken symlinks left behind by `pnpm prune`
+    pnpm store prune
```

Comment:
I wonder why this touches the contents of node_modules. The help page mentions that it only really cleans up the store